index.bs

```diff
@@ -1,11 +1,11 @@
 <pre class=metadata>
-Title: Web Audio API
+Title: Web Audio API 1.1
 Shortname: webaudio
-Level: 1
+Level: 1.1
 Group: audiowg
-Status: ED
+Status: FPWD
 ED: https://webaudio.github.io/web-audio-api/
-TR: https://www.w3.org/TR/webaudio/
+TR: https://www.w3.org/TR/webaudio-11/
 Favicon: favicon.png
 Previous Version: https://www.w3.org/TR/2021/CR-webaudio-20210114/
```

Comment:
Should we add https://www.w3.org/TR/2021/REC-webaudio-20210617/ to previous versions too?

Reply:
Good thought! Done.